### ISSUE TITLE
"Alerted" status is not marked along with "Blocked"

### ISSUE THREAD

**Issue by korgull** [SecurityRiskAdvisors/VECTR#6]
Outcomes can only be "Detected" or "Blocked", which is mitigated by the fact that the operator can note that the Red Team action caused an Alert.  However, there is nothing at a high level that notes this activity.  In my organization, that would create a bit of heartburn.  We look for the action to be "Detected" or "Not Detected".  If it was detected, then it stands to reason that the SOC follows through to stop the bleeding.  Yes, we do have cases such as communication being blocked by IronPort or BlueCoat with no alarm, but my leadership prefers to know if 1) the activity was seen and 2) how quick did we respond.

It is a matter of preference, so, this is comment is more of a suggestion that an issue.  It's an issue for me, but not necessarily for you.

**Comment by thebleucheese** [comment]
Just to make sure I'm looking at the right place(<base_url>/sra-purpletools-webui/app/#/app/assessmentGroup/campaigns/campaign) and then clicking one of the test cases to edit.

For outcomes on the Edit Test Case Screen in the "Blue Team Details" panel I see 4 possible outcomes that can be checked - TBD, Blocked, Detected, and NotDetected (the last one is wrapped to a new line for me on a smaller monitor which can make it easier to miss).  By any chance do you have a screenshot of the area in particular where you're only seeing two outcomes?

Just as a note - On the different reporting screens you can filter by Outcome, using all 4 outcomes or only the ones that you need to see the current assessment status.

**Comment by korgull** [comment]
Hi,
I see multiple outcomes, it would just be nice to have the alert trigger shows in the log or somewhere we can see without digging too far.

I mentioned it as a preference later, but I think the outcome should be “Detected” or “Not Detected”, with a section below as to wether or not it was also blocked (or not detected, but blocked).  It may be a preference and I only have experience with one organization I work for.

**Comment by korgull** [comment]
Hi,
I see multiple outcomes, it would just be nice to have the alert trigger shows in the log or somewhere we can see without digging too far.

I mentioned it as a preference later, but I think the outcome should be “Detected” or “Not Detected”, with a section below as to wether or not it was also blocked (or not detected, but blocked).  It may be a preference and I only have experience with one organization I work for.

**Comment by thebleucheese** [comment]
This is a suggestion worth investigating - let's brainstorm keeping detected/not detected and blocked/not blocked as separate data points for a test case.  This may have implications on the granularity of that data.  For instance, would we need to apply that detection credit and blocking credit to two separate tools?

**Comment by thebleucheese** [comment]
Just an update, we started UI mockups and work on a revamp for the data we capture around detection.

**Comment by SolAngelus** [comment]
+1 to requesting an "Alerted" box to go along with the Blocked and Detected boxes.

**Comment by SRAPSpencer** [comment]
Good news! Our latest release has new outcome complexity in line with this feature request. Feel free to download and check it out. 

https://github.com/SecurityRiskAdvisors/VECTR/releases/tag/ce-8.4.2
